src/sentry/conf/server.py

```diff
@@ -1040,7 +1040,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     # Enables getting commit sha from git blame for codecov.
     "organizations:codecov-commit-sha-from-git-blame": False,
     # Enables automatically deriving of code mappings
-    "organizations:derive-code-mappings": False,
+    "organizations:derive-code-mappings": True,
```

Comment:
We already have this feature enabled for *all* organizations in our SaaS (option set at 100%):
https://github.com/getsentry/getsentry/blob/c05c17d3386b2ae43a52edae2d79c9b7dd33d0d7/getsentry/features.py#L1155-L1158

In case I'm wrong. You may want to deploy soon after the PR merge and keep an eye for new issues.

We also have [this](https://app.datadoghq.com/dashboard/drp-ufq-pzc/team-issue-experience-downstream-consumers?from_ts=1678904920392&to_ts=1679509720392&live=true) DD dashboard to check for the queue size..

Reply:
Right, it's enabled for all orgs, but only runs 1/1000th of the time. My understanding was that that was in aid of doing a partial rollout, and that now we're doing a full rollout, that roll of the dice won't happen anymore. (That's what https://github.com/getsentry/getsentry/pull/9905 does.) Is that not correct?

Reply:
Sorry, which value is set to 1? The thing we're modding and dividing is the org id.

https://github.com/getsentry/getsentry/blob/feb5dc7d248fe5a65eeecd7b5841c4fb18c5061a/getsentry/features.py#L1155-L1158

```Python
        if (
            organization.id % 100000
        ) / 100000 < self.cached_ga_rollout_percentage:  # (0.00001, float value between 0 and 1)
            return True
```

Reply:
That's correct.

Reply:
Great - going to merge it in, then!